support-frontend/assets/helpers/user/__tests__/userReducerTest.ts

```diff
@@ -1,109 +1,127 @@
 // ----- Imports ----- //
-import {
-	Canada,
-	GBPCountries,
-	UnitedStates,
-} from '../../internationalisation/countryGroup';
+import type { Action } from '../userActions';
 import { createUserReducer } from '../userReducer';
 // ----- Tests ----- //
 describe('user reducer tests', () => {
+	const nonExistentAction = {
+		type: 'DOES_NOT_EXIST',
+	};
+
 	it('should return the initial state', () => {
-		expect(createUserReducer(GBPCountries)(undefined, {})).toMatchSnapshot();
+		expect(
+			createUserReducer()(undefined, nonExistentAction as Action),
+		).toMatchSnapshot();
 	});
```

Comment:
`createUserReducer` no longer takes a countryGroupId as a parameter for the initial state following [this pr](https://github.com/guardian/support-frontend/pull/1945)